server/src/main/java/org/candlepin/util/Util.java

```diff
@@ -143,6 +147,13 @@ public static Date toDate(String dt) {
         }
     }
 
+    public static Date toDate(OffsetDateTime dt) {
+        if (dt == null) {
+            return null;
+        }
+        return Date.from(dt.toInstant());
+    }
+
     public static <T> T assertNotNull(T value, String message) {
```

Comment:
This is unrelated to your PR, but can we put a TODO on this to axe this entirely? I don't think anyone knows it exists, and it doesn't provide any real value other than saving two lines of code.

Reply:
Done.